perun-base/src/main/java/cz/metacentrum/perun/core/api/Group.java

```diff
@@ -116,6 +116,19 @@ public void setShortName(String shortName) {
 		}
 	}
 
+	/**
+	 * Get whole parent group prefix.
+	 * For example for group A:B:C return "A:B"
+	 * For group A return empty String (there is no prefix in such case)
+	 *
+	 * @return whole prefix (without the last ':' character)
+	 */
+	public String getPrefix() {
```

Comment:
This will affect all (de)serialization of the Group object. It you make it utils method, we can have this change as a quick fix in the nearest release. Otherwise it must me updated on other places too and we must notify other parties before deployment.

Reply:
Logic was changed.